pytest_splunk_addon/standard_lib/data_models/Alerts.json

```diff
@@ -1,75 +1,128 @@
-            },
-            {
-                "name": "type",
-                "type": "required",
-                "expected_values":[
-                    "alarm",
-                    "alert",
-                    "event",
-                    "task",
-                    "warning"
-                ],
-                "comment":"The message type." 
-            }
-        ],
-        "child_dataset": []
-    }]
-}
\ No newline at end of file
+    "objects": [
+        {
+            "name": "Alerts",
+            "tags": [
+                [
+                    "alert"
+                ]
+            ],
+            "fields_cluster": [],
+            "search_constraints": "tag=alert",
+            "fields": [
+                {
+                    "name": "app",
+                    "type": "required",
+                    "comment": "The application involved in the event, such as win:app:trendmicro, vmware, nagios."
+                },
+                {
+                    "name": "body",
+                    "type": "required",
+                    "comment": "The body of a message."
+                },
+                {
+                    "name": "description",
+                    "type": "optional",
+                    "comment": "The description of the alert event."
+                },
+                {
+                    "name": "dest",
+                    "type": "required",
+                    "comment": "The destination of the alert message, such as an email address or SNMP trap. You can alias this from more specific fields, such as dest_host, dest_ip, or dest_name."
+                },
+                {
+                    "name": "dest_type",
+                    "type": "optional",
+                    "comment": "The type of the destination object, such as instance, storage, firewall."
+                },
+                {
+                    "name": "id",
+                    "type": "required",
+                    "comment": "The unique identifier of a message."
+                },
+                {
+                    "name": "mitre_technique_id",
+                    "type": "optional",
+                    "comment": "The MITRE ATT&CK technique ID of the alert event."
+                },
+                {
+                    "name": "signature",
+                    "type": "optional",
+                    "comment": "A human-readable signature name."
+                },
+                {
+                    "name": "signature_id",
+                    "type": "optional",
+                    "comment": "The unique identifier or event code of the event signature."
+                },
+                {
+                    "name": "severity",
+                    "type": "required",
+                    "expected_values": [
+                        "critical",
+                        "high",
+                        "medium",
+                        "low",
+                        "informational"
```

Comment:
`unknown` is missing here?

Reply:
This was not added, reason can be found here: https://github.com/splunk/pytest-splunk-addon/pull/595#discussion_r840602600

Reply:
this is in line with our 'don't map null values' policy